cmd/mb-main.go

```diff
@@ -29,7 +29,7 @@ var (
 	mbFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "region",
-			Value: "us-east-1",
+			Value: "",
```

Comment:
@harshavardhana, overriding region inside client-s3 so that default is not changed